src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -447,6 +447,135 @@ JSON example:
+| PulseSequenceType                 | REQUIRED                                                                                                                        | [string][]                              | General description of the pulse sequence used for the scan. Allowed values are any combination of 2D or 3D acquisition, with the read-out type, separated by an underscore: for example “2D_EPI”, “3D_GRASE”, “3D_spiral”.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| EchoTime                          | REQUIRED                                                                                                                        | [number][] or [array][] of [numbers][]  | The echo time (TE) for the acquisition, specified in seconds. Specify either one number for the total time-series, or provide an array of numbers with different values of each volume. Corresponds to DICOM Tag 0018, 0081 `Echo Time` (please note that the DICOM term is in milliseconds not seconds).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
```

Comment:
This seems to conflict with existing approaches to multi-echo data (i.e., using the echo entity and storing the data in separate files). Do the separate echoes need to be in the same file?